### PR TITLE
LRQA-54502 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LRQA-54502
AppBuilder#ReuseExistingFormViewFields

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -1849,6 +1849,90 @@ definition {
...
+		FormViewBuilder.validateFormMultipleSelectionField(
+			fieldLabel = "Multiple 1",
+			fieldName = "Hobbies");
+
+		FormViewBuilder.editFieldInObject(
+			fieldLabel = "Testing Changed Name",
+			fieldName = "Nickname");
+
+		FormViewBuilder.addFieldToObject(fieldName = "Text");
+
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hey @nkkwon , thanks for the improvements, I saw that you added a macro to validate if the required option was properly set to the added field in the Form View. However, the most important here is to make sure this property was persisted when reusing this field adding it to a new form view. So, could you please add this validation at the end of the script too? Thanks!

**Comment by nkkwon** [comment]
@alexcandidos I updated the test, please take a look and let me know if I have to change something!

**Comment by alexcandidos** [comment]
Hi @nkkwon , I did some improvements in this PR and I just forwarded it to BChan, you can see the changes here: https://github.com/brianchandotcom/liferay-portal/pull/86207. Thanks !

**Pull request closed without merging**